### PR TITLE
V1 job submit

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/cloud/issues/34

### CODE REVIEW

File: paddlecloud/paddlejob/views.py [hunk 1]
```diff
@@ -60,4 +74,14 @@ def delete(self, request, format=None):
...
-        return utils.simple_response(200, "OK")
+        username = request.user.username
+        namespace = notebook.utils.email_escape(username)
+        obj = json.loads(request.body)
+        jobname = obj.get("jobname")
+        if not jobname:
+            return utils.simple_response(500, "must specify jobname")
+        # FIXME: options needed: grace_period_seconds, orphan_dependents, preconditions
+        u_status = client.BatchV1Api().delete_namespaced_job(jobname, namespace, {})
+        pserver_name = "-".join(jobname.split("-")[:-1])
```
Comment (Yancey1989) [inline paddlecloud/paddlejob/views.py]: maybe var `pserver_name` is unuseful.

File: go/paddlecloud/get.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		fmt.Fprintf(os.Stderr, "error getting jobs: %v\n", err)
+	}
+	var respObj interface{}
+	err = json.Unmarshal(respBody, &respObj)
+	if err != nil {
+		return err
+	}
+	items := respObj.(map[string]interface{})["items"].([]interface{})
+	if len(items) >= 0 {
+		fmt.Printf("NUM\tNAME\tSUCC\tFAIL\tSTART\tCOMP\tACTIVE\n")
```
Comment (Yancey1989) [inline go/paddlecloud/get.go]: `fmt.Printf("NUM\tNAME\tSUCC\tFAIL\tSTART\tCOMP\tACTIVE\n") `
=>
`fmt.Printf(strings.Join("\t",string[]{"NUM", "NAME", "SUCCESS", "FAIL", "START", "COMPELTED", "ACTIVE"}))`

Comment (typhoonzero) [inline go/paddlecloud/get.go]: Seems no need to allocate a temporary `string[]`?

Comment (Yancey1989) [inline go/paddlecloud/get.go]: Use `Join`, we don't need to write so much `\t`, but it doesn't matter, everyone is OK 😀

**Pull request merged**